doc/glossary.rst

```diff
@@ -76,13 +76,31 @@ MNE-Python core terminology and general concepts
         down to a change of the ``first_samp`` attribute to know when cropped data
         was acquired.
 
+    forward solution
+        The forward solution or forward model (abbr. ``fwd``) is a linear
+        operator capturing the relationship between each dipole location in the
+        :term:`source space` and the expected pattern of activity on the
+        sensors corresponding to that dipole (AKA, the "lead fields"). Forward
```

Comment:
```suggestion
         measured by the sensors (AKA, the "lead fields"). Forward
```
(Nitpick) Is it necessary to mention lead fields here? In my mind, LFs are source-space sensitivity maps for a given `fwd`, so more like a visualisation of it than an "AKA"?

Reply:
yes agreed. So it's more the rows of the fwd.

@mshamalainen please educate us if need be :)

Reply:
@cjayb at the code sprint last month, there was specific mention of "lead fields" being notably absent from MNE documentation, which can pose a barrier to people who are used to that terminology from using other software.  That's why I added it here.  But it's entirely possible that I'm slightly mistaken about what "lead fields" means... I thought based on [this Brainstorm page](https://neuroimage.usc.edu/brainstorm/Tutorials/HeadModel?highlight=%28lead%29%7C%28field%29#Forward_model) that "lead field" was basically synonymous with "forward model":

> In Brainstorm, we call the outcome of this modeling step a "head model", a.k.a. forward model, leadfield matrix or gain matrix in the MEG/EEG literature.

Reply:
@drammock OK, it's been discussed, I yield :)@britta-wstnr did mention that you'd spoken about it, and I agree to the extent that it's important not to confuse newcomers. How about

(the "lead field" or "gain" matrix is a component of the forward solution)

or something to that extent? (`fwd['sol']['data']`)

Reply:
"lead field matrix" to me is synonymous with "forward matrix" and "gain matrix", and it appears to be for @agramfort [at least back in 2011](https://www.hindawi.com/journals/cin/2011/923703/) :)

> For each of these instances, the result of the forward problem is expressed as a lead field, that is, the matrix representing the linear relation between sources and measurements, a.k.a. “Gain matrix.”

The confusion perhaps comes from describing a lead field as one column of the forward matrix (how all sensors see one source space dipole) rather than a single row of the forward matrix (how one sensor sees all source space dipoles). So perhaps scrap the row/col terminolology in favor of a more generic linear-relationship phrasing, and we should be good here.

Reply:
OK, I confess now that I'm thoroughly confused.  If our definition starts out by saying "the forward solution is a linear operator" (AKA a matrix) then in what sense is the "lead field matrix" simply a *component* of the forward solution (as opposed to being *the same thing as* the forward solution)?  Is there some equivocation going on here between the concept of "forward solution" versus what we store in a `mne.forward.forward.Forward` object (which includes all kinds of other stuff)?

Reply:
A lead field for a single sensor is one row of the gain/forward/lead field matrix. Lead field matrix, forward matrix, and gain matrix are synonyms.

Reply:
OK, but if the "forward solution" is a linear operator, then "forward solution" == "forward matrix" == "lead field matrix"?
```
The forward solution (abbr. ``fwd``) is a linear operator capturing the
relationship between each dipole location in the :term:`source space`
and the corresponding field distribution measured by the sensors (AKA, the
"lead field matrix").
```
OK?

Reply:
Yeah, that looks good to me

Reply:
ok for me too